src/main/java/io/cdap/plugin/dlp/SensitiveRecordFilter.java

```diff
@@ -261,6 +293,30 @@ private DlpServiceSettings getSettings() throws IOException {
     @Description("Error handling of record")
     private String onError;
 
+    @Macro
+    @Name("dlp-override-enabled")
+    @Description("Use custom DLP service endpoint")
+    private Boolean dlpOverrideEnabled;
+
+    @Macro
+    @Name(DLP_HOST)
+    @Nullable
+    @Description("DLP host")
```

Comment:
maybe provide an example of what the host format looks like?

Reply:
I'll add some samples of hostnames.